lib/generators/connector/templates/stylesheets/variables.sass

```diff
@@ -36,6 +38,17 @@ $small:                          11px
 $normal:                         13px
```

Comment:
I know it's not yours but the var name is not ideal (does not make sense without the Text Size comment)